lib/charms/layer/kubernetes_common.py

```diff
@@ -996,7 +996,7 @@ def get_sandbox_image_uri(registry):
     return "{}/pause:3.4.1".format(registry)
 
 
-def configure_kubelet(dns_domain, dns_ip, registry, taints=None):
+def configure_kubelet(dns_domain, dns_ip, registry, taints=None, has_xcp=False):
```

Comment:
looks like there's a chance we'll need to add more optional `configure_kubelet` arguments. 

how about:



```suggestion
def configure_kubelet(dns_domain, dns_ip, registry, **kwargs):
    taints=kwargs.get("taints", None)
    has_xcp=kwargs.get("has_xcp", False)
```

Reply:
nah, ignore this.  I understand now this is just silly and unnecessary.